controllers/topolvm_storageclass.go

```diff
@@ -106,6 +129,11 @@ func getTopolvmStorageClasses(lvmCluster *lvmv1alpha1.LVMCluster) []*storagev1.S
 				"csi.storage.k8s.io/fstype": TopolvmFilesystemType,
 			},
 		}
+		// reconcile will pick up any existing LVMO storage classes as well
+		if defaultStorageClassName == "" || defaultStorageClassName == storageClass.Name {
```

Comment:
won't ` defaultStorageClassName == "" ` condition be enough even if the reconciler picks up existing lvmo storage classes ?

Reply:
This is to handle a scenario where multiple VGs may be supported.